private/rules/jvm_import.bzl

```diff
@@ -42,6 +48,8 @@ def _jvm_import_impl(ctx):
 
     compilejar = ctx.actions.declare_file("header_" + injar.basename, sibling = injar)
     args = ctx.actions.args()
+    args.add("-jar")
```

Comment:
Nit (so feel free to ignore): using `all_all(["-jar", add_jar_manifest_entry_jar])` groups the logical unit into a single line which is a little easier to understand when reading the code.

Reply:
Done.